src/db/logger.cr

```diff
@@ -0,0 +1,30 @@
+require "logger"
+
+module DB
+  # :nodoc:
+  class Logger
+    @logger : Proc(String, Void)?
+    @enabled = false
+    @default_logger = ::Logger.new(STDOUT)
+
+    def config(@logger)
+    end
+
+    def logging=(@enabled : Bool)
+    end
+
+    def log(query : String, *args)
+      return if !@enabled
+      if args && args.size > 0
+        log = %{QUERY "#{query}" WITH PARAMS: #{args.join(", ")}}
```

Comment:
How about `"Running query #{query.inspect} with params: #{args.join(", ")}"`. I think human readable is better than shouting in all caps here.